source/administration/legacy-upgrade.md

```diff
@@ -1,17 +1,109 @@
-# Legacy Upgrade Guide
+# Upgrading from an Unsupported Version
 
-This is a legacy upgrade guide for servers running Mattermost version v2.2.0 and earlier. If you are upgrading a server running Mattermost v3.0.x or later, please refer to our most [current upgrade guide](https://docs.mattermost.com/administration/upgrade.html).
+These instructions apply only if you are upgrading from an unsupported version of Mattermost to the latest version. Only the current released version and the two previous versions are supported.
```

Comment:
`Only the current released version and the two previous versions are supported.` 

^for older releases, but we support upgrading from any 3.X.X to any 3.X.X (as long as you're upgrading to a newer release)